packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.0.14 - 2019-03-26
+* Added event timeline to memory profiler to track DevTool's Snapshot and Reset events.
+* Timeline CPU renamed to UI, janky defined as UI duration + GPU duration > 16 ms.
+* Timeline frame chart removed 8 ms highwater line, only 16 ms highwater line, display 2 traces ui/gpu (instead of 4). Janky frames will have a red glow.
+* Flame chart colors use a different set of palettes and timeline is sticky.
+* Properly handle tabs that aren't supported.
```

Comment:
This is a little vague (it might not be clear what "properly handle", "tabs" or "supported" mean). Maybe properly handle = disable, tabs = sections? features?, not supported = not supported for the current/connected application.

(that said, I'm not sure how detailed we're trying to be with release notes, I think many people probably won't see them since we just silently update in the background right now - maybe we should try and coordinate with tooling updates and link here from their release notes or something?)

Reply:
On the micro level updated to:

* Properly disable features that aren't supported for the connected application.

On the macro question, would be nice if the implementer (PR submitter) added the feature/change to the CHANGELOG at time of PR.

Reply:
Yeah, that makes sense - and something to call out in PRs if it's missing (since it'll be easy to forget).